core/dbt/config/project.py

```diff
@@ -709,7 +715,6 @@ def to_project_config(self, with_packages=False):
                 "exposures": self.exposures,
                 "vars": self.vars.to_dict(),
                 "require-dbt-version": [v.to_version_string() for v in self.dbt_version],
-                "config-version": self.config_version,
```

Comment:
is this just removing some dead code? or are we actively removing support for config-version in dbt_project.yml with this change?

Reply:
We removed support for config-version sometime last year, but a few things were still hanging around.  It still exists in the class that validates because we don't want to cause a validation error, but there's no point in passing it along.